.github/workflows/awstest.yml

```diff
@@ -0,0 +1,31 @@
+name: nf-core AWS test
+# This workflow is triggered on PRs to the master branch.
+# It runs the -profile 'test_full' on AWS batch
+
+on:
+  pull_request:
```

Comment:
```suggestion
  push:
```

Reply:
We decided to go for push, as tests would be triggered multiple times if pushes are added to PRs to master. We want to test only once all the other tests pass.

Reply:
sure